packages/web-components/fast-element/karma.conf.cjs

```diff
@@ -0,0 +1,162 @@
+const path = require('path');
+
+const basePath = path.resolve(__dirname);
+
+const commonChromeFlags = [
+  '--no-default-browser-check',
+  '--no-first-run',
+  '--no-sandbox',
+  '--no-managed-user-acknowledgment-check',
+  '--disable-background-timer-throttling',
+  '--disable-backing-store-limit',
+  '--disable-boot-animation',
+  '--disable-cloud-import',
+  '--disable-contextual-search',
+  '--disable-default-apps',
+  '--disable-extensions',
+  '--disable-infobars',
+  '--disable-translate',
+];
```

Comment:
should we run with experimental-web-platform-features?

Reply:
I'm not sure to be honest. Are we using any experimental features? Maybe TrustedTypes?

Reply:
I believe focus-visible is still behind the flag, I'm not sure if we want to run with it on or off.

Reply:
Off is probably fine for now, I'll resolve